docs/index.css

```diff
@@ -5,18 +5,13 @@ body {
   line-height: 2rem;
   text-transform: initial;
   letter-spacing: initial;
-  font-weight: 400;
   color: #212b36;
-  font-family: -apple-system, 'BlinkMacSystemFont', 'San Francisco', 'Roboto',
-    'Segoe UI', 'Helvetica Neue', sans-serif;
-}
+  font-family: -apple-system, 'BlinkMacSystemFont', 'San Francisco', 'Roboto', 'Segoe UI', 'Helvetica Neue', sans-serif; }
 
 @media (min-width: 40em) {
   html,
   body {
-    font-size: 1.4rem;
-  }
-}
```

Comment:
Where are these trailing bracket changes coming from? Can they be reverted?

Reply:
Yep - that was due to the fact sewing-kit didn't format css files.